home_page/src/main/java/com/trempel/home_page/ui/HomePageViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.trempel.home_page.ui
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import javax.inject.Inject
+
+
+class HomePageViewModel @Inject constructor() : ViewModel() {
+
+    val searchQueryData = MutableLiveData<String>()
+    private val _onSearch = MutableLiveData<String>()
+    val onSearch: LiveData<String> get() = _onSearch
```

Comment:
I would prefer to rename it in such a way that the word `submit` is in the title.